index.html

```diff
@@ -0,0 +1,205 @@
+          </h2>
+          <div class="paramain">
+              <p class="headpart-text bg-light">
+                We are organizing a concert for fundraising to feed people who
+                are affected by the ongoing drought.Donate to support humanitarian work around Kenya. 
+              </p>
+          </div>
+
+          <h4 class="head-h4">LET'S CONNECT</h4>
+          <h4 class="head-h4 head-h4-strong" ><strong>FROM (MOND 8AM) ~ (SUN 17AM)</strong></h4>
+          <h4 class="head-h4 head-h4-strong">Lets fundraise</h4>
+        </section>
+     
+      <!-- Card section Mobile & Desktop -->
+      <section id="program" class="bg-dark paddings">
+        <h4 class="text-center">Main Program</h4>
+        <div class="orangeColor guideBar marginXAuto mx-auto"></div>
+        <div class="container pb-3">
+          <div class="row justify-content-md-center">
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/registration.webp" class="card-img-top bg-white" alt="icon">                
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Registration</h5>
+                  <p class="card-text">All the guests will register for attendance</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/food-items.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of food items</h5>
+                  <p class="card-text">All food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/sleep_pillow_blanket-256.webp" class="bg-white card-img-top" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Collection of non-food items</h5>
+                  <p class="card-text">All non-food items will be collected and stored awaiting distribution.</p>
+                </div>
+              </div>
+            </div>
+            <div class="col-md-6 col-lg-3">
+              <div class="card follow-up">
+                <img src="./image/money.webp" class="card-img-top bg-white" alt="icon">
+                <div class="follow-up-body">
+                  <h5 class="follow-up-title">Cash fundraising</h5>
+                  <p class="card-text">Cash, money transfer and cheque will be collected.</p>
+                </div>
+              </div>
+            </div>
+            <a href="#" class="justify-content-md-center text-white">SEE THE WHOLE PROGRAM</a>
+            </div>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] As the previous reviewer mentioned, kindly make sure this section looks identical to the original design

|Your design|Original design|
|-|-|
|![Screenshot_20220924_152821](https://user-images.githubusercontent.com/25227022/192103741-4eafae97-96d5-4ded-a7a8-e68c61fe6729.png)|![Screenshot_20220924_153156](https://user-images.githubusercontent.com/25227022/192103727-59f9a141-bee2-4b9d-a1bb-f37f3ed7d5bc.png)|